src/InsertionsClient.Console/ConsoleApp/Program.cs

```diff
@@ -137,14 +144,20 @@ private static int Main(string[] args)
             {
                 ignoredPackages = InsertionConstants.DefaultDevUxTeamPackages;
             }
+
+            if(!string.IsNullOrWhiteSpace(BuildFilterString) && !InputLoading.LoadBuildFilter(BuildFilterString, out buildFilter))
+            {
+                ShowErrorHelpAndExit("Failed to parse build filters from input string: " + BuildFilterString);
+            }
             
             UpdateResults results = api.UpdateVersions(
                     manifestFiles,
                     DefaultConfigFile,
                     whitelistedPackages,
                     ignoredPackages,
                     FeedAccessToken,
-                    PropsFilesRootDirectory
+                    PropsFilesRootDirectory,
```

Comment:
design question...do you think we have reached the point where UpdateVersions shold be overloaded to accept a richer data structure than the n-many method parameters?

Reply:
Yes, but I have exhausted all the time I have for this feature. So I will need to postpone this to next feature implementation.

Reply:
makes sense